Public/Src/App/Bxl/Args.cs

```diff
@@ -290,6 +290,12 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                         OptionHandlerFactory.CreateBoolOption(
                             "debugScript",
                             opt => frontEndConfiguration.DebugScript = opt),
+                        OptionHandlerFactory.CreateOption(
+                            "globalUnsafePassthroughEnvVars",

```

Comment:
>globalUnsafePassthroughEnvVars [](start = 29, length = 30)

Please order it alphabetically so that it is easy to find.

Reply:
The argument list is now out of order, I will place these two new arguments in a relatively more reasonable position. But will not reorder the whole thing in this PR.